CMakeLists.txt

```diff
@@ -23,6 +23,8 @@ option(LIB_PROTO_MUTATOR_EXAMPLES "Enable examples building" ON)
 option(LIB_PROTO_MUTATOR_DOWNLOAD_PROTOBUF
```

Comment:
Would you be able to send OSS-Fuzz patch as well?

Reply:
@vitalybuka after merge definitely, yes :+1:

Reply:
@vitalybuka here you go: https://github.com/google/oss-fuzz/pull/13073